exercises/pythagorean-triplet/src/Triplet.hs

```diff
@@ -1,7 +1,10 @@
 module Triplet (isPythagorean, mkTriplet, pythagoreanTriplets) where
 
-isPythagorean = error "You need to implement this function."
+isPythagorean :: (Int, Int, Int) -> Bool
```

Comment:
I don't think we should stop things here because of that. We don't have anything ready yet.